src/certificates/certificates-card/certificate-details/CertificateDetails.test.jsx

```diff
@@ -0,0 +1,122 @@
+import { Provider, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { render, waitFor, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+
+import initializeStore from '../../../store';
+import { MODE_STATES } from '../../data/constants';
+// import { setMode } from '../../data/slice';
+import messages from '../messages';
+import CertificateDetails from './CertificateDetails';
+
+let store;
+const courseId = 'course-v1:edX+DemoX+Demo_Course';
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useDispatch: jest.fn(),
+}));
+jest.mock('@edx/frontend-platform/i18n', () => ({
```

Comment:
[question]: In the file `src/certificates/certificates-card/CertificatesCard.test.jsx` you added indents after each `jest.mock`. Is this trend worth sticking with?